### ISSUE TITLE
seems deprecated in favor of dcv; should it be deleted/renamed mir-cv-deprecated?

### ISSUE THREAD

**Issue by timotheecour** [libmir/mir-cv#4]

**Comment by ljubobratovicrelja** [comment]
It's not deprecated, just never taken care of. Idea was to have separate repo with lower level CV algorithms, implemented in betterC style, with optional C API headers. In fact, DCV was supposed to be relying on mir-cv as dependency, providing higher level API for those lower level algorithms. #3 was first step towards that, but I got caught up in too many stuff to explain here, and unfortunately never been able to finish it since.

**Comment by 9il** [comment]
Hi @ljubobratovicrelja. Sorry about #3 stop. Lets move it forward. We would be able to optimise kernels later.

**Comment by ljubobratovicrelja** [comment]
> We would be able to optimise kernels later.

I agree. I'll take a look at it, see what more can be easily done and I'll try and do it in the next few weeks.

**Comment by wilzbach** [comment]
Dumb questions: why don't you simply have a `mir-cv` subpackage in the `dcv` repo? That would save you a lot of troubles with keeping things in sync and people who are interested in just the betterC stuff, can do e.g.

```
dependency "mir-cv:mir" version="~>0.9.0"
```

**Comment by ljubobratovicrelja** [comment]
@9il had this idea of a separate repo. I thought it was OK, since there might be people who would only be interested in this project, and not in DCV. But anyways, because of the lack of action, I think that's the last of our problems at the moment. :) IMO we could always relocate the project to DCV in future if we feel like it.

**Comment by 9il** [comment]
Yes, probably we can have a DUB sub package.
